### PR TITLE
Add max_1080_video_quality:

### PR DESCRIPTION
**Pull request by catduckgnaf**
max_1080_video_quality: as a common preset. 

  max_1080_video_quality:
    ytdl_options:
      format: "(bv*[height<=1080]+bestaudio/best[height<=1080])"
      merge_output_format: "mp4"
      remux_video: "mp4"

### CODE REVIEW

**Comment by jmbannon** [comment]
remux_video I don't think is a yt-dlp arg. Let's use the file_convert plugin instead

**Comment by catduckgnaf** [comment]
How about we take out the bottom two lines? We should not care about the format, if we are having the most vanilla preset for 1080p possible?

**Comment by jmbannon** [comment]
let's take the remux out, the merge I think is fine to keep since it will have to merge anyways. May as well use the most compatible codec type

**Comment by jmbannon** [comment]
Last ask: let's add documentation for it just like the best_video_quality preset: https://github.com/jmbannon/ytdl-sub/blob/master/docs/presets.rst?plain=1#L148

**Comment by catduckgnaf** [comment]
> remux_video I don't think is a yt-dlp arg. Let's use the file_convert plugin instead
I did that because best_video_quality also has remux_video, should we remove it there as well?

File: docs/presets.rst [hunk 1]
```diff
@@ -153,6 +153,15 @@ it into an MP4 container:
 
 * ``best_video_quality``
 
+
+Limit maximum quality to 1080P (No 4K)
```
Comment (jmbannon) [inline docs/presets.rst:157]: Let's rename this title to `Max 1080p`

Comment (catduckgnaf) [reply docs/presets.rst:157]: That is for the existing preset that was there.  not my addition.

Best Video Quality
^^^^^^^^^^^^^^^^^^

Add the following preset to download the best available video and audio quality, and remux
it into an MP4 container:

* ``best_video_quality``


Limit maximum quality to 1080P (No 4K)
^^^^^^^^^^^^^^^^^^

Add the following preset to download the best available (but not greater then 1080P video) and audio quality
The output container MP4:

* ``max_1080p``

Comment (jmbannon) [reply docs/presets.rst:157]: I'm saying let's make the title you added simpler, from 'Limit maximum quality to 1080P (No 4K' to just 'Max 1080p'

File: docs/presets.rst [hunk 2]
```diff
@@ -153,6 +153,15 @@ it into an MP4 container:
 
 * ``best_video_quality``
 
+
+Max 1080p
+^^^^^^^^^^^^^^^^^^
+
+Add the following preset to download the best available (but not greater then 1080P video) and audio quality
```
Comment (jmbannon) [inline docs/presets.rst:160]: Last nit: "(but not greater than) 1080p video and audio quality"

Comment (catduckgnaf) [reply docs/presets.rst:160]: If I want to nit, audio quality can't be 1080p. so having the 1080p limit in parentheses I believe would be the most accurate.
It is still the best audio quality. Only the video quality is limited.

Comment (jmbannon) [reply docs/presets.rst:160]: True, how about 'best available audio and video quality, with the video not greater than 1080p'

**Review by catduckgnaf — COMMENTED** [review]
Changes made

**Pull request merged**